src/import/load.js

```diff
@@ -95,13 +95,22 @@ function xhrHasResponse(request) {
     request.responseText; // '' on error
 }
 
-function xhr(url, callback) {
+function xhrSetHeader(request, headers) {
+  if (headers && request.setRequestHeader) {
+    for (var name in headers) {
+      request.setRequestHeader(name, headers[name]);
+    }
+  }
+}
+
+function xhr(opts, callback) {
+  opts = typeof opts === 'string' ? { url: opts } : opts;
```

Comment:
Why switch to an options hash here? `xhr` is only called from the `load` method with a string url argument. Or are you overriding `sanitizeUrl` to return an object?